packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -133,9 +133,11 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
   }
 
   private _renderInnerText(): JSX.Element {
+    const nativeProps = getNativeProps(this.props, divProperties);
```

Comment:
should this be `spanProperties` just because you're mixing it into a span? (I wonder this aloud without checking to see if we have spanProperties.. it might be the case that divProperties and spanProperties are the same thing)

Reply:
Good point. `spanProperties` doesn't exist, but `htmlElementProperties` is probably a good alternative.

Reply:
If we're mixing in nativeProps, can/should we remove/deprecate the ariaLabel prop from MessageBar.types?

Reply:
@leddie24 good call, yes we should! The ariaLabel prop actually doesn't even get used anywhere right now lol.